bokeh/models/glyphs.py

```diff
@@ -124,7 +132,7 @@
 # General API
 #-----------------------------------------------------------------------------
 
-class AnnularWedge(XYGlyph):
+class AnnularWedge(LineGlyph, FillGlyph):
```

Comment:
Removing `XYGlyph` is incorrect, because you are removing vital properties and inheritance chain. Those `{Line,Fill,Hatch}Glyph` classes should be treated as mixins, so in the end the signature should look like this:
```py
class AnnularWedge(XYGlyph, LineGlyph, FillGlyph):
```
This is fine, because Python supports multiple inheritance. This obviously applies to all other changes.

Reply:
OK, that makes sense - thanks.